### DISCUSSION TITLE
如何用graph scope进行分页

### DISCUSSION THREAD

**Discussion by altqxdd** [alibaba/GraphScope#2449] [Q&A] [answered]
想要按照按顶点的某个属性排序，如下：
g.V().order().by('address', asc).limit(10)

现在想取按address排序后第10～20个，或者20～30个，做一个分页的功能，有什么解决办法吗

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by siyuan0322** [comment]
Maybe you could use `.next()`
For example:
```python
t = g.V().order().by('address', asc)
t.next(10)
t.next(10)
```

**Comment by altqxdd** [comment]
> 
next 好像并不支持
![image](https://user-images.githubusercontent.com/125848466/220106601-bdb56c62-f7cc-48ed-870c-c9f36b2a8974.png)

**Comment by siyuan0322** [comment, marked as answer]
Please use bytecode mode, as follows:
```python
g = interactive.traversal_source()
t = g.V().order().by('address', asc)
print(t.next(10))
print(t.next(10))
```
I'm not an expert of gremlin, perhaps @shirly121  could show how to use pagination in script mode.
